src/fuzzy_finder/fuzzy_finder.rs

```diff
@@ -30,7 +30,9 @@ pub fn run(makefile: Makefile) {
     }
 }
 
-fn get_preview_command(file_paths: Vec<String>) -> String {
+fn get_preview_command(mut file_paths: Vec<String>) -> String {
+    // workaround for https://stackoverflow.com/questions/15432156/display-filename-before-matching-line
```

Comment:
I think the comments you wrote here (https://github.com/kyu08/fzf-make/issues/53#issuecomment-1684872018) will be of great help to future readers of this part (myself included).
So how about adding that link to the code comments like below?

```suggestion
    // workaround for https://stackoverflow.com/questions/15432156/display-filename-before-matching-line
    // For more information, see https://github.com/kyu08/fzf-make/issues/53#issuecomment-1684872018
```

Reply:
Sounds reasonable!